### PR TITLE
Create initial thanos-mixin with querier alerts

### PR DESCRIPTION
**Pull request by metalmatze**
This is a very first draft of 3 alerts I could come up with looking at the Thanos Querier metrics.

/cc @brancz @aditya-konarde @bwplotka

### CODE REVIEW

File: jsonnet/thanos-mixin/alerts/querier.libsonnet [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          },
+          {
+            alert: 'ThanosQuerierRangeLatencyHigh',
+            annotations: {
+              message: 'Thanos Querier has a 99th percentile latency of {{ $value }} seconds for instant queries.',
+            },
+            expr: |||
+              histogram_quantile(0.99,
+                sum(thanos_query_api_range_query_duration_seconds_bucket{%(thanosQuerierSelector)s}) by (le)
+              ) > 1
```
Comment (brancz) [inline jsonnet/thanos-mixin/alerts/querier.libsonnet]: I'm fairly certain that we'll hit this often until we introduce the Cortex style caching, but I like the ambition :wink:

**Comment by brancz** [comment]
lgtm

File: jsonnet/thanos-mixin/alerts/querier.libsonnet (hunk 1 above)
Comment (metalmatze) [reply jsonnet/thanos-mixin/alerts/querier.libsonnet]: 😂 🚀
What would you suggest? 5s?

Comment (brancz) [reply jsonnet/thanos-mixin/alerts/querier.libsonnet]: Let’s keep like this and see how it goes

**Pull request merged**